book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly ensure that the value of rentals cannot only be read but also changed by adding it to ```attr_accessor``` as this would help in future projects.